src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -28,8 +28,18 @@
         line-height: 20px;
     }
 
+    &.bcs-is-scrolled::before {
+        position: absolute;
+        // position it behind bcs-content-header
+        top: -60px;
+        right: 20px;
+        left: 20px;
+        height: 60px;
+        box-shadow: 0 4px 12px rgba(0, 0, 0, .1);
```

Comment:
Is this a standard box shadow? I don't see it being used elsewhere.

Reply:
I don't think so. It's the shadow that appears under header when Feed is scrolled down.

Reply:
For the box shadow let's extract the values into variables similar to how it's done it other places. Look at line 67 as a guid e  e.g.    
box-shadow: 0 0 $bdl-grid-unit * 4 $bdl-gray-30;